lesson07/assignment/vue-cli/src/components/Searchbar.vue

```diff
@@ -0,0 +1,95 @@
+<template>
+   <div class="search-wrapper">
+      <label>
+         <h2>search</h2>
+      </label>
+      <input type="text"  placeholder="This doesn't work yet"/>
+      <div class='side'>
+         <ul class="side__list" data-title="Central America">
+            <div  v-for="(card, idx) in decks" v-bind:key="idx">
+               <div  v-if='card.area == "Central America"'>
+                  <!--couldn't not figure out more clever way to get unique area values-->
```

Comment:
An alternative would be to subdivide the array into regions using `filter`. It would actually be less efficient code, because it would loop through the list twice, once to create those filtered arrays, then again through the filtered arrays. But it might be preferable for other reasons, like readability. 
```
let ca = decks.filter((obj)=>{return obj.area == 'Central America'})
```
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter